bbcu/libs/logins.js

```diff
@@ -251,7 +251,7 @@ class Login {
 					// look for connection offer that has the nonce from the qr code
 					const watcher = new InboundNonceWatcher(
 						this.agent,
-						InboundNonceWatcher.REQUEST_TYPES.CONNECTION | InboundNonceWatcher.REQUEST_TYPES.VERIFICATION,
+						InboundNonceWatcher.REQUEST_TYPES.CONNECTION,
 						this.qr_code_nonce, 30, 3000);
 					const nonce_item = await watcher.start();
 					if (nonce_item && nonce_item.state) {
```

Comment:
I think that these bitwise or's '|' need to stay.  Did you see them causing problems, is that why you removed them?  I think they are there for cases where a connection may already exist and thus there will be no incoming connection offer to look for.

Reply:
Yep okay, that was my bad I didn't fully understand what that was doing, didn't understand what "5" meant. Added it back and accounted for it in the error message.